model/src/main/kotlin/AnalyzerResult.kt

```diff
@@ -51,13 +51,7 @@ data class AnalyzerResult(
     // Do not serialize if empty to reduce the size of the result file. If there are no errors at all,
     // [AnalyzerResult.hasErrors] already contains that information.
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
-    val errors: SortedMap<Identifier, List<OrtIssue>> = sortedMapOf(),
-
-    /**
-     * A map that holds arbitrary data. Can be used by third-party tools to add custom data to the model.
-     */
-    @JsonInclude(JsonInclude.Include.NON_EMPTY)
-    val data: CustomData = emptyMap()
+    val errors: SortedMap<Identifier, List<OrtIssue>> = sortedMapOf()
```

Comment:
Commit message: Just say "...auto-generated functions...", besides `equals` it also affects `hashCode`, `componentN`, `toString`, and `copy`.

Reply:
Done.